packages/design-system/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livechat/design-system",
-  "version": "0.7.0",
+  "version": "0.7.1",
```

Comment:
I am missing updated version in package-lock.json, please run lerna bootstrap to make sure you have all packages up-to-date

Reply:
I just did and there was nothing to commit